src/main/java/com/google/devtools/build/remote/client/LogParserUtils.java

```diff
@@ -58,6 +62,7 @@ public ParamException(String message) {
 
   public LogParserUtils(String filename) {
     this.filename = filename;
+    this.jsonString = "[";
```

Comment:
This seems really awkward, why are you assigning this here? I don't see any reason to maintain the `print` result in a local member for each call...